crates/rooch/src/commands/account/commands/verify.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use async_trait::async_trait;
+use clap::Parser;
+use fastcrypto::{
+    hash::Sha256,
+    secp256k1::{Secp256k1PublicKey, Secp256k1Signature},
+    traits::ToFromBytes,
+};
+use rooch_types::{
+    crypto::{RoochSignature, Signature},
+    error::{RoochError, RoochResult},
+};
+
+/// Verify a signature
+#[derive(Debug, Parser)]
+pub struct VerifyCommand {
+    /// A signature for verify
+    #[clap(short = 's', long)]
+    signature_hash: String,
```

Comment:
signature not signature_hash

Reply:
There needs a value parser to convert them from output to the input.